src/js/tracks/html-track-element.js

```diff
@@ -0,0 +1,86 @@
+import * as browser from '../utils/browser.js';
+import document from 'global/document';
+import EventTarget from '../event-target';
+
+export const NONE = 0;
```

Comment:
We've generally been putting all exports at the bottom of a file.
Though, I meant more like:
```js
HTMLTrackElement.NONE = 0;
HTMLTrackElement.LOADING = 1;
// ...
```
Sorry I wasn't more clear earlier.